style.css

```diff
@@ -1,7 +1,639 @@
+    color: #585858;
+    font-size: 20px;
+    font-weight: 900;
+    outline: none;
+    padding-bottom: 13px;
+  }
+
+  .participants {
+    align-items: center;
+    background-color: #405dbb;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    margin-bottom: 33px;
+    max-height: 442px;
+    max-width: 320px;
+  }
+
+  .wrap-info {
+    color: #fff;
+    font-weight: 900;
+    margin: 0;
+    padding-top: 37px;
+    padding-left: 0;
+  }
+
+  .wrap-info__title {
+    align-self: center;
+    font-size: 20px;
+    margin: 0;
+    margin-bottom: 4px;
+    max-height: 23px;
+    max-width: 152px;
+    padding-left: 85px;
+    padding-right: 83px;
+    text-align: center;
+  }
+
+  .wrap-info__paragraph {
+    align-self: center;
+    font-size: 10px;
+    padding-bottom: 0;
+    padding-left: 52px;
+    padding-right: 53px;
+    margin: 0 0 21px 0;
+    max-height: 24px;
+    max-width: 215px;
+    text-align: center;
+  }
+
+  .wrap-info__pic {
+    align-self: center;
+    margin-bottom: 28px;
+    max-height: 222px;
+    max-width: 260px;
+    padding-left: 30px;
+    padding-right: 30px;
+  }
+
+  .button__request {
+    margin-bottom: 38px;
+  }
+
+  .speakers {
+    align-items: center;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+  }
+
+  .speakers__title {
+    font-size: 20px;
+    font-weight: 500;
+    text-align: center;
+    margin-bottom: 28px;
+    margin-top: 0;
+    max-width: 199px;
+  }
+
+  .photos {
+    margin-bottom: 80px;
+    padding: 0 calc(30px - 2 * (30px - (30 * 100% / 320)));
+  }
+
+  .photos__item:first-child {
+    align-self: center;
+    height: 260px;
+    margin: 0 0 20px 0;
+    width: 260px;
+  }
+
+  .photos__item:nth-child(2) { display: none; }
+
+  .photos__item:nth-child(3) { display: none; }
+
+  .photos__item:nth-child(4) { display: none; }
+
+  .photos__item:nth-child(5) { display: none; }
+
+  .photos__item:nth-child(6) { display: none; }
+
+  .photos__item:nth-child(7) { display: none; }
+
+  .photos__item:nth-child(8) { display: none; }
+
+  .photos__item:nth-child(9) {
+    height: 10px;
+    margin-right: 18px;
+    padding-left: 97px;
+    visibility: visible;
+    width: 10px;
+  }
+
+  .photos__item:nth-child(10) {
+    height: 10px;
+    margin-right: 18px;
+    visibility: visible;
+    width: 10px;
+  }
+
+  .photos__item:nth-child(11) {
+    height: 10px;
+    visibility: visible;
+    width: 10px;
+  }
+
+  .footer {
+    background-color: #0e1e71;
+    color: #e1e6e9;
+    display: flex;
+    flex-direction: row;
+    font-size: 8px;
+    font-style: normal;
+    font-weight: 900;
+    max-height: 92px; 
+    max-width: 320px;
+    padding: 0 calc(30px - 2 * (30px - (30 * 100% / 320)));
+  }
+
+  .location {
+    display: flex;
+    flex-direction: row;
+    color: #e1e6e9;
+    margin-right: 61px;
+    padding-right: 0px;
+    padding-top: 16px;
+  }
+
+  .location__pic {
+    height: 9px;
+    margin-right: 3px; 
```

Comment:
Codacy found an issue: [Expected newline after ";" in a multi-line declaration block (declaration-block-semicolon-newline-after)](https://app.codacy.com/gh/WebHeroSchool/IFW.2/pullRequest?prid=6289680)